docs/modules/ROOT/pages/release-notes.adoc

```diff
@@ -1,31 +1,35 @@
-= 5.4 Release Notes
+= 5.5 Release Notes
 
-In this release, we introduced the xref:user-code-deployment.adoc[User Code Deployment] feature, which allows you to deploy custom and domain classes to Hazelcast members. We also introduced support for xref:wan-replication.adoc[WAN replication].
+In this release, we introduced the Custom Resources for xref:topic-configuration.adoc[Topic], xref:multimap-configuration.adoc[Multimap] and xref:replicatedmap-configuration.adoc[Replicated Map] data structure support. We also introduced EntryListener support for Map CR.
 
 == New features
 
-* Added support for WAN replication (https://github.com/hazelcast/hazelcast-platform-operator/pull/226[#226])
-* Added support for the `EntryProcessor` interface (https://github.com/hazelcast/hazelcast-platform-operator/pull/334[#334])
-* Added supported for the `MapStore`/`MapLoader` interfaces (https://github.com/hazelcast/hazelcast-platform-operator/pull/291[#291])
-* Added support for the Executor service (https://github.com/hazelcast/hazelcast-platform-operator/pull/300[#300])
+* Add support for MultiMap CR (https://github.com/hazelcast/hazelcast-platform-operator/pull/367[#367])
+* Add support for Topic CR (https://github.com/hazelcast/hazelcast-platform-operator/pull/371[#371])
+* Add support for ReplicatedMap CR (https://github.com/hazelcast/hazelcast-platform-operator/pull/394[#394])
+* Add support for CronHotBackup CR (https://github.com/hazelcast/hazelcast-platform-operator/pull/362[#362])
+* Add support for restoring from HotBackup resource name (https://github.com/hazelcast/hazelcast-platform-operator/pull/401[#401])
+* Make Jet engine configurable from Hazelcast CR (https://github.com/hazelcast/hazelcast-platform-operator/pull/457[#457])
+* Add Map Entry listener to get notified when a map entry is added, removed, updated, evicted, or expired (https://github.com/hazelcast/hazelcast-platform-operator/pull/408[#408])
```

Comment:
```suggestion
* When a map entry is added, removed, updated, evicted, or expired, the Map Entry listener is notified (https://github.com/hazelcast/hazelcast-platform-operator/pull/408[#408])
```

Reply:
@angelasimms I'm not sure about it. 
We intentionally want to mention that it is a new feature. With the suggested change, it looks like we just changed/updated functionality. But before, we don't have a Map Entry listener.

Reply:
@cheels would this be better. 

Added a Map Entry listener that is notified when a map entry is added, removed, updated, evicted, or expired (https://github.com/hazelcast/hazelcast-platform-operator/pull/408[#408])

Reply:
@angelasimms Yes, much better! Thanks!